QUEST_LV_0200.tsv

```diff
@@ -1186,15 +1186,15 @@ QUEST_LV_0200_20150317_001185	I feel hungry no matter how much I eat..{nl}Hey..
 QUEST_LV_0200_20150317_001186	Give me some food, quickly.
 QUEST_LV_0200_20150317_001187	I am part of the priests who were sent to release the curse of Naktis. {nl}We must study and collect data in order to release the curse of Naktis. {nl}But I have to find the bishop who said he'd protect the Cathedral after the Day of Divine Tree. {nl}
 QUEST_LV_0200_20150317_001188	Will you help me find him?
-QUEST_LV_0200_20150317_001189	Oh no.. The bishop returned as a corpse.. {nl}Thank you for your help.
+QUEST_LV_0200_20150317_001189	Oh no... The bishop returned as a corpse... {nl}Thank you for your help.
 QUEST_LV_0200_20150317_001190	I need to bury the bishop but there are too many monsters around.
-QUEST_LV_0200_20150317_001191	I shouldn't have left him in the Cathedral..
+QUEST_LV_0200_20150317_001191	I shouldn't have left him in the Cathedral...
 QUEST_LV_0200_20150317_001192	I received your help again. Thank you very much.
 QUEST_LV_0200_20150317_001193	I'm worried about the whereabouts of the bishop.
-xQUEST_LV_0200_20150317_001194	Use Naktis' monsters.. {nl}But how can we use them.. {nl}Aha! We can turn into Naktis' monsters from the sanctuary and make the other monsters destroy it.
-QUEST_LV_0200_20150317_001195	There are materials in the sanctuary with instructions on how to make medicine that turns you into a monster. {nl}Find those materials.
+xQUEST_LV_0200_20150317_001194	Use Naktis' monsters... {nl}But how can we use them... {nl}Aha! We can turn into Naktis' monsters from the sanctuary and make the other monsters destroy it.
+QUEST_LV_0200_20150317_001195	There are materials in the sanctuary with instructions on how to make a medicine that turns you into a monster. {nl}Find those materials.
 QUEST_LV_0200_20150317_001196	Now it's time to tell Naktis' monsters where Mayvern's device is located.
-QUEST_LV_0200_20150317_001197	I fell while holding the materials and it went under that device. {nl}That thing keeps moving up and down making it difficult to get it out.
+QUEST_LV_0200_20150317_001197	I fell while holding the materials, and they fell under that device. {nl}It keeps moving up and down making it difficult to get them out.
```

Comment:
You always need to use a comma when you have two complex clauses joined together. Or, more simply, you cannot join two complete sentences without a comma. My word choice of "fell" being used twice is questionable. Since "materials" is a plural noun, "it went" is incorrect. Slid may work better but it doesn't quite fit the sentence either.